central/sensor/service/connection/sensorevents.go

```diff
@@ -80,6 +82,7 @@ func (s *sensorEventHandler) addMultiplexed(ctx context.Context, msg *central.Ms
 		if err := s.pipeline.Reconcile(ctx, s.reconciliationMap); err != nil {
 			log.Errorf("error reconciling state: %v", err)
 		}
+		s.initSyncMgr.Remove(s.cluster.GetId())
```

Comment:
It might be good to have a comment that explains why this specific sensor message type is considered to be the end of a sensor init sync.